### PR TITLE
Block domain fixes

### PR DESCRIPTION
**Pull request by ArikDay**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CIAC-8219

## Description
Fixed playbooks visualization, fixed bugs found and performed minor enhancements. All changes are detailed in the Release Notes.
Deprecating Block Domain - Generic

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/TrendMicroApex/Playbooks/playbook-Block_Domain_-_Trend_Micro_Apex_One.yml [hunk 1]
```diff
@@ -198,24 +201,29 @@ tasks:
...
-  value:
-    simple: www.test.com
+  value: {}
   required: false
   description: The Domain to block.
   playbookInputQuery:
+- key: Expiration
+  value: {}
+  required: false
+  description: 'The UTC expiration date and time of the suspicious object, for example: 2020-01-25T09:00:00Z'
```
Comment (ShirleyDenkberg) [inline Packs/TrendMicroApex/Playbooks/playbook-Block_Domain_-_Trend_Micro_Apex_One.yml:225]:
```suggestion
  description: 'The UTC expiration date and time of the suspicious object, for example: 2020-01-25T09:00:00Z.'
```

File: Packs/TrendMicroApex/Playbooks/playbook-Block_Domain_-_Trend_Micro_Apex_One_README.md [hunk 2]
```diff
@@ -2,31 +2,41 @@ This playbook blocks domains using Trend Micro Apex One.
...
 
 ## Playbook Inputs
+
 ---
 
 | **Name** | **Description** | **Default Value** | **Required** |
 | --- | --- | --- | --- |
-| Domain | The Domain to block. | www.test.com | Optional |
+| Domain | The Domain to block. |  | Optional |
+| Expiration | The UTC expiration date and time of the suspicious object, for example: 2020-01-25T09:00:00Z |  | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/TrendMicroApex/Playbooks/playbook-Block_Domain_-_Trend_Micro_Apex_One_README.md:31]:
```suggestion
| Domain | The domain to block. |  | Optional |
| Expiration | The UTC expiration date and time of the suspicious object, for example: 2020-01-25T09:00:00Z. |  | Optional |
```

File: Packs/TrendMicroApex/ReleaseNotes/2_0_5.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Block Domain - Trend Micro Apex One
+
+- Removed the unnecessary 'Domain' input default value.
+- Added the input 'Expiration' to support expiration date and time for blocked domains.
```
Comment (ShirleyDenkberg) [inline Packs/TrendMicroApex/ReleaseNotes/2_0_5.md:7]:
```suggestion
- Added the input 'Expiration' to support the expiration date and time for blocked domains.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_Domain_-_Generic_v2.yml [hunk 4]
```diff
@@ -409,6 +415,11 @@ inputs:
     Tag to assign Domain to the External Dynamic List.
     sub-playbook: Block Domain - External Dynamic List
   playbookInputQuery:
+- key: Expiration
+  value: {}
+  required: false
+  description: "The UTC expiration date and time of the suspicious object, for example: 2020-01-25T09:00:00Z\nProducts: \nTrend Micro Apex One\nProofpoint Threat Response"
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_Domain_-_Generic_v2.yml:421]:
```suggestion
  description: "The UTC expiration date and time of the suspicious object, for example: 2020-01-25T09:00:00Z.\nProducts: \nTrend Micro Apex One\nProofpoint Threat Response"
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_Domain_-_Generic_v2_README.md [hunk 5]
```diff
@@ -10,38 +10,49 @@ Supported integrations for this playbook:
...
 ## Playbook Inputs
+
 ---
 
 | **Name** | **Description** | **Default Value** | **Required** |
 | --- | --- | --- | --- |
 | Domain | The Domain to block. |  | Optional |
 | DomainBlackListID | The Domain List ID to add the Domain to.<br/>product: Proofpoint Threat Response |  | Optional |
+| Tag | Tag to assign Domain to the External Dynamic List.<br/>sub-playbook: Block Domain - External Dynamic List |  | Optional |
+| Expiration | The UTC expiration date and time of the suspicious object, for example: 2020-01-25T09:00:00Z<br/>Products: <br/>Trend Micro Apex One<br/>Proofpoint Threat Response |  | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_Domain_-_Generic_v2_README.md:47]:
```suggestion
| Expiration | The UTC expiration date and time of the suspicious object, for example: 2020-01-25T09:00:00Z.<br/>Products: <br/>Trend Micro Apex One<br/>Proofpoint Threat Response |  | Optional |
```

File: Packs/ProofpointThreatResponse/Playbooks/playbook-Block_Domain_-_Proofpoint_Threat_Response.yml [hunk 6]
```diff
@@ -225,6 +230,11 @@ inputs:
   required: false
   description: The ID of the block list to block the domain in.
   playbookInputQuery:
+- key: Expiration
+  value: {}
+  required: false
+  description: 'The UTC expiration date and time of the suspicious object, for example: 2020-01-25T09:00:00Z'
```
Comment (ShirleyDenkberg) [inline Packs/ProofpointThreatResponse/Playbooks/playbook-Block_Domain_-_Proofpoint_Threat_Response.yml:236]:
```suggestion
  description: 'The UTC expiration date and time of the suspicious object, for example: 2020-01-25T09:00:00Z.'
```

**Comment by ShirleyDenkberg** [comment]
@AdiPeret Doc review completed.

**Review by AdiPeret — APPROVED** [review]

**Pull request merged**